### PR TITLE
[TST] Show logging output during integration tests

### PR DESCRIPTION
**Pull request by tsalo**
Closes None.

Changes proposed in this pull request:

- Show info-level logging output during integration tests.

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -18,13 +18,13 @@ unittest:
 	@py.test --skipintegration --cov-append --cov-report term-missing --cov=tedana tedana/
 
 three-echo:
-	@py.test --cov-append --cov-report term-missing --cov=tedana -k test_integration_three_echo tedana/tests/test_integration.py
+	@py.test --log-cli-level=20 --cov-append --cov-report term-missing --cov=tedana -k test_integration_three_echo tedana/tests/test_integration.py
```
Comment (emdupre) [inline Makefile:21]:
```suggestion
	@py.test --log-cli-level=INFO --cov-append --cov-report term-missing --cov=tedana -k test_integration_three_echo tedana/tests/test_integration.py
```

File: Makefile [hunk 2]
```diff
@@ -18,13 +18,13 @@ unittest:
...
 
 four-echo:
-	@py.test --cov-append --cov-report term-missing --cov=tedana -k test_integration_four_echo tedana/tests/test_integration.py
+	@py.test --log-cli-level=20 --cov-append --cov-report term-missing --cov=tedana -k test_integration_four_echo tedana/tests/test_integration.py
```
Comment (emdupre) [inline Makefile:24]:
```suggestion
	@py.test --log-cli-level=INFO --cov-append --cov-report term-missing --cov=tedana -k test_integration_four_echo tedana/tests/test_integration.py
```

File: Makefile [hunk 3]
```diff
@@ -18,13 +18,13 @@ unittest:
...
 
 five-echo:
-	@py.test --cov-append --cov-report term-missing --cov=tedana -k test_integration_five_echo tedana/tests/test_integration.py
+	@py.test --log-cli-level=20 --cov-append --cov-report term-missing --cov=tedana -k test_integration_five_echo tedana/tests/test_integration.py
```
Comment (emdupre) [inline Makefile:27]:
```suggestion
	@py.test --log-cli-level=INFO --cov-append --cov-report term-missing --cov=tedana -k test_integration_five_echo tedana/tests/test_integration.py
```

File: Makefile [hunk 4]
```diff
@@ -18,13 +18,13 @@ unittest:
...
 
 t2smap:
-	@py.test --cov-append --cov-report term-missing --cov=tedana -k test_integration_t2smap tedana/tests/test_integration.py
+	@py.test --log-cli-level=20 --cov-append --cov-report term-missing --cov=tedana -k test_integration_t2smap tedana/tests/test_integration.py
```
Comment (emdupre) [inline Makefile:30]:
```suggestion
	@py.test --log-cli-level=INFO --cov-append --cov-report term-missing --cov=tedana -k test_integration_t2smap tedana/tests/test_integration.py
```

**Comment by jbteves** [comment]
Oops sorry @emdupre didn't realize you'd already been asked to review already 🤦‍♂️

**Pull request merged**